### DISCUSSION TITLE
How to save the video stream from Scrcpy?

### DISCUSSION THREAD

**Discussion by ZHENGYUN01** [yume-chan/ya-webadb#423] [Q&A]
Well, this git hub works well for me, but I want to save my screen data directly, how can i do this on the basis of this git hub.

thx for your time.

**Comment by yume-chan** [comment]
What does the "screen data" mean?

**Reply by ZHENGYUN01** [reply]
@yume-chan 
it means H264data of android phone screen collected by scrcpy.
I want to save screen data as a video when scrcpy runs.

**Reply by ZHENGYUN01** [reply]
hi, yume-chan,
could you please to spend some minutes to view my question?
thx for you time.
